examples/pascal_multilabel_with_datalayer_tools.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
```

Comment:
I think this would be at home as `caffe/examples/pycaffe/tools.py` to make it more clearly related to pycaffe examples.